gammapy/visualization/utils.py

```diff
@@ -126,9 +130,15 @@ def patch_func(
 
 def plot_contour_line(ax, x, y, **kwargs):
     """Plot smooth curve from contour points"""
+    xf = x
+    yf = y
+
     # close contour
-    xf = np.append(x, x[0])
-    yf = np.append(y, y[0])
+    rel_tol = 1.e-2
```

Comment:
I think the equivalency here should be exact, no? Why do we need the tolerance?

Reply:
I do not know what iminuit will do exactly... This is why I put some tolerance. As you prefer...

Reply:
A priori minuit simply closes the contour the same way. A tolerance should not be necessary

Reply:
Changed